binders/kafka-binder/spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/properties/KafkaBinderConfigurationProperties.java

```diff
@@ -665,4 +671,15 @@ public KafkaProducerProperties getExtension() {
 
 	}
 
+	public static class Metrics {
```

Comment:
No strong feelings on how to name this, any thoughts?

Reply:
`Metrics` sounds too generic. It might be better to qualify with some prefixes.

Reply:
I think its already namespaced by the enclosing class and I followed the example of the `Transaction` class in the same file that doesn't use any more prefixes, see [here](https://github.com/spring-cloud/spring-cloud-stream/blob/66552a91495e06f95e300c0e4d256be3c3eff310/binders/kafka-binder/spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/properties/KafkaBinderConfigurationProperties.java#L482)

Reply:
Ok, that makes sense. Fair enough.